recipes/hazelcast-cpp-client/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "4.2.0":
+    url: "https://github.com/hazelcast/hazelcast-cpp-client/archive/master.zip"
```

Comment:
A head of a branch is not immutable and will not produce a constant hash.
Did you add 4.2.0 to check for an imminent release?

Reply:
Yes, that's why the PR is still in draft state. I will change it when the release tag is published.

Reply:
@madebr, PR is ready now. Can you review?